src/reggae/dub/interop.d

```diff
@@ -153,7 +153,11 @@ private from!"reggae.dub.info".DubInfo _getDubInfo(T)(auto ref T output,
                         gDubInfos[config].packages[0].dflags ~= " -unittest";
                     }
 
-                    callPreBuildCommands(options, gDubInfos[config]);
+                    try
+                        callPreBuildCommands(output, options, gDubInfos[config]);
+                    catch(Exception e) {
+                        output.log("Error calling prebuild commands: ", e.msg);
+                    }
```

Comment:
What's the rationale for allowing failure here?

Reply:
We have no control over external commands. Do you mean the exception should be handled elsewhere?

Reply:
Sure, we don't have control, but just because we don't have control doesn't mean we want to carry on even if it fails, surely?

Reply:
https://github.com/atilaneves/reggae/pull/71